arroyo-sql/src/expressions.rs

```diff
@@ -3617,6 +3628,14 @@ pub struct RustUdafExpression {
     ret_type: TypeDef,
 }
 impl RustUdafExpression {
+    pub(crate) fn name(&self) -> String {
+        self.name.clone()
+    }
+
+    pub fn expressions(&mut self) -> Vec<&mut Expression> {
+        self.args.iter_mut().map(|(_, e)| e).collect()
+    }
```

Comment:
maybe better to return this as an iter so that you're not creating a vec that's not needed

```suggestion
    pub fn expressions(&mut self) -> impl Iterator<Item = &mut Expression> {
        self.args.iter_mut().map(|(_, e)| e)
    }
```

Reply:
Done